src/utils/timeSeriesResultsByOption.js

```diff
@@ -98,13 +104,10 @@ function getMonthlyResult (
     pmt = PMT(mortgageRate / COMPOUND_FREQUENCY, 1, lastResult.loanAmt)
   }
 
-  // Everything else can be invested
-  const investmentPMT = lastResult.budget - pmt
```

Comment:
The main issue is that you were using the previous `investmentPMT` which was already adjusted